### PR TITLE
added functions to manage bc online environment update windows

### PR DESCRIPTION
**Pull request by JeremiasBr**
**Added two scripts for managing bc online environment update windows.**

Scripts:

- **Saas\Get-BcEnvironmentUpdateWindow.ps1:** Function for getting Update Window on a Business Central online environment
- **Saas\Set-BcEnvironmentUpdateWindow.ps1:** Function for setting Update Window on a Business Central online environment.

Hopefully, this time around, I didn't miss anything..

### CODE REVIEW

**Comment by pri-kise** [comment]
Normally you should add a few lines to releasenotes.txt as well.

File: Saas/New-BcEnvironment.ps1 [hunk 1]
```diff
@@ -51,7 +51,7 @@ function New-BcEnvironment {
         $bcAuthContext = Renew-BcAuthContext -bcAuthContext $bcAuthContext
         Wait-BcEnvironmentsReady -environments @($environment) -bcAuthContext $bcAuthContext -apiVersion $apiVersion -applicationFamily $applicationFamily
```
Comment (freddydk) [inline Saas/New-BcEnvironment.ps1:53]: I also ran into this - it has been fixed in the main repo now

**Comment by freddydk** [comment]
PR looks good (haven't tested though) - please change from draft to ready for review when you are ready (and have resolved the conflict in releasenotes)

**Comment by JeremiasBr** [comment]
One thing I noticed - I had to copy "Create-SaasUrl" in "BcContainerHelper.psd1" to make the function get recognized. I didnt include this change in any commit, because the psd1's will be created automatically.
I believe the issue was a missing Export-ModuleMember in "SaaSHelperFunctions.ps1", which I have added.

File: SaaSHelperFunctions.ps1 [hunk 2]
```diff
@@ -27,4 +27,6 @@
     }
 
     return $bcAuthContext, $headers, $endPointURL
-}
\ No newline at end of file
+}
+
+Export-ModuleMember -Function Create-SaasUrl
```
Comment (freddydk) [inline SaaSHelperFunctions.ps1:32]: The Create-SaasUrl should definitely not be exported - it is in the helperfunctions, which should be available to the functions inside containerhelper

**Review by freddydk — CHANGES_REQUESTED** [review]

File: SaaSHelperFunctions.ps1 (hunk 2 above)
Comment (freddydk) [reply SaaSHelperFunctions.ps1:32]: And it shouldn't be in the .psd1 either.
All other functions use this function and it is not needed for them.

Comment (JeremiasBr) [reply SaaSHelperFunctions.ps1]: okay, sorry - then I misunderstood. I have adjusted the files accordingly.

File: BC.SaasHelper.ps1 [hunk 3]
```diff
@@ -1,10 +1,12 @@
 # SaaS specific functions
+. (Join-Path $PSScriptRoot "SaaSHelperFunctions.ps1")
```
Comment (freddydk) [inline BC.SaasHelper.ps1:2]: This one also should not be here - the SaaSHelperFunctions should be included elsewhere - all the other functions use it as well

**Review by freddydk — CHANGES_REQUESTED** [review]
Sorry, one more.

**Pull request merged**